src/Helper/Names/DefaultNamesHelper.php

```diff
@@ -12,6 +13,16 @@ class DefaultNamesHelper implements NamesHelperInterface
 {
     use HelperTrait;
 
+    public function format(string $name): string
```

Comment:
There is no information of what this method takes as an argument: `$name` is ambiguous. We should at least replace that with `$manifestName`

Reply:
Also, maybe we should rename `format()` method to something like `fullName`